2019/_data/speakers.json

```diff
@@ -176,6 +176,17 @@
       "url": "infoSpeaker.html?ref=e4dbc8f86240f1f78408c1e967137aa5d8824024",
       "homepage": "https://github.com/madsop/talks",
       "twitter": "@madsopheim"
+    },
+    {
+      "enabled": "1",
+      "name": "Victor Rentea",
+      "description": "Independent Trainer & Coach, Lead Architect at IBM Romania",
```

Comment:
He adds `Software Craftsman` to his twitter and LinkedIn account. I think we should add it too.